### ISSUE TITLE
Removal of Rewards, Referal & Youtube sync (what??)

### ISSUE THREAD

**Issue by danger89** [lbryio/lbry-desktop#7386]
**Bug**

@jessopb recently **removed** several _key features_ from the LBRY desktop app. Which includes the rewards section, referral features and YouTube sync during registration and much more...

I have no clue what is going on here, but I think this is a huge mistake! 

@tzarebczan I hope there is a very very good reason behind this?

- https://github.com/lbryio/lbry-desktop/commit/514bc0a273069d0e5a04608cfa05037522b23059
- https://github.com/lbryio/lbry-desktop/commit/6ba1fafaa07fee0915075e3033fba5af8cd4c6d9
- https://github.com/lbryio/lbry-desktop/commit/5e09de5f9433fd8a8dd32e9e00fe3ea05ff93001

**To Reproduce**
1. Account Menu
2. You will notice the missing menu items like 'Rewards' and 'Refferal' items missing from this menu

**Expected behavior**

I do not expect that features are getting removed on purpose! LBRY App should be as feature risk as the lbry/odysee website at least, if not more.

**Screenshots**

**Before:**
![image](https://user-images.githubusercontent.com/628926/147890064-85411f69-91c8-4b17-91f3-d49d1ce99bfa.png)

**After:**
![image](https://user-images.githubusercontent.com/628926/147889961-11718039-6149-4fef-bde9-6cefaeb5dfe4.png)

**System Configuration**

 v0.52.1-alpha.1

**Comment by eggplantbren** [comment]
I agree with you, but this was done on purpose. One of the occasional baffling things you'll get to enjoy by following LBRY and Odysee :-)

**Comment by danger89** [comment]
> I agree with you, but this was done on purpose. One of the occasional baffling things you'll get to enjoy by following LBRY and Odysee :-)

Yea, I created a fork: https://github.com/lbry-foss/lbry-desktop Wanna join?

**Comment by kodxana** [comment]
From my point rewards were part of centralized services when the goal of LBRY App is to be less dependent from LBRY Inc API. Removing rewards and referal links is good thing cause that means LBRY Inc is working on better replacement for rewards. Mayby we finally get paid hosting?

**Comment by danger89** [comment]
> Removing rewards and referal links is good thing cause that means LBRY Inc is working on better replacement for rewards. Mayby we finally get paid hosting?

These are just assumptions for now. I didn't show any posts or updates from LBRY Inc. about a new reward system. They even did not communicate the removal beforehand.

**Comment by kauffj** [comment]
We're removing features from LBRY Desktop that are use LBRY Inc. in some way. Rewards and sync continue to be offered by @OdyseeTeam
